applications/GeoMechanicsApplication/tests/cpp_tests/test_T_micro_climate_flux_condition.cpp

```diff
@@ -467,14 +447,13 @@ KRATOS_TEST_CASE_IN_SUITE(CalculateLocalSystemForThermalMicroClimateCondition3D8
     p_condition->CalculateLocalSystem(lhs_matrix, rhs_vector, r_model_part.GetProcessInfo());
 
     auto expected_lhs_matrix = Matrix{8, 8, 0.0};
-    expected_lhs_matrix <<= 5.26894, 1.75631, 2.63447, 1.75631, -5.26894, -7.02525, -7.02525, -5.26894,
-            1.75631, 5.26894, 1.75631, 2.63447, -5.26894, -5.26894, -7.02525, -7.02525,
-            2.63447, 1.75631, 5.26894, 1.75631, -7.02525, -5.26894, -5.26894, -7.02525,
-            1.75631, 2.63447, 1.75631, 5.26894, -7.02525, -7.02525, -5.26894, -5.26894,
-            -5.26894, -5.26894, -7.02525, -7.02525, 28.101, 17.5631, 14.0505, 17.5631,
-            -7.02525, -5.26894, -5.26894, -7.02525, 17.5631, 28.101, 17.5631, 14.0505,
-            -7.02525, -7.02525, -5.26894, -5.26894, 14.0505, 17.5631, 28.101, 17.5631,
-            -5.26894, -7.02525, -7.02525, -5.26894, 17.5631, 14.0505, 17.5631, 28.101;
+    expected_lhs_matrix <<= 5.26894, 1.75631, 2.63447, 1.75631, -5.26894, -7.02525, -7.02525,
```

Comment:
This might be nice to revert and add the tags to ignore this for formatting

Reply:
Done.